### ISSUE TITLE
 Documentation link doesn't work

### ISSUE THREAD

**Issue by keh9mark** [bcakmakoglu/vue-flow-pathfinding-edge#2]
Hello. Documentation link doesn't work https://vueflow.dev/docs/addons/pathfinding

**Comment by bcakmakoglu** [comment]
Yes, this package is currently in the process of being moved into the the vue flow repository.
Docs will be added back soon, sorry for the issue 😅
